### PR TITLE
✨ Get Cosmos Address from Keplr instead Ledger

### PR DESCRIPTION
**Pull request by shellyYG**

### CODE REVIEW

File: components/StepCosmos.vue [hunk 1]
```diff
@@ -103,6 +124,32 @@ export default {
...
+        console.error(err);
+      }
+    },
+    async signInWithPlatform(platform, opt = {}) {
+      this.platform = platform; // leave this line for future platform expansion.
+      const { source } = opt;
+      this.signInWithCosmosWallet(source);
+    },
+    onClickUseKeplr() {
+      this.signInWithPlatform('cosmosWallet', { source: 'keplr' });
```
Comment (williamchong) [inline components/StepCosmos.vue:151]: Just
```
this.cosmosAddress = await Keplr.getWalletAddress();
this.submitCosmosAddress();
```
would do

File: util/api/index.js [hunk 2]
```diff
@@ -16,4 +16,4 @@ export const apiGetPendingCosmosMigration = (address, ethMigrationTxHash) => api
   `/migrate/pending/cosmos/${address}?eth_tx=${ethMigrationTxHash}`,
 );
 
-export const apiGetCosmosBalance = (address) => api.get(`/migrate/cosmos/${address}`);
+export const apiGetCosmosBalance = (address) => api.get(`/bank/balances/${address}`);
```
Comment (williamchong) [inline util/api/index.js:19]: Why this change?

**Review by williamchong — CHANGES_REQUESTED** [review]

File: util/api/index.js (hunk 2 above)
Comment (shellyYG) [reply util/api/index.js]: oh! actually this is not needed. Fixed here: https://github.com/likecoin/migrate-to-likecoin-chain/pull/57/commits/2665d6c41041aa2422b6d3186fc120e2be2c02f3

File: server/config/config.js [hunk 3]
```diff
@@ -4,6 +4,7 @@ export const FIRESTORE_CONFIG_ROOT = '';
 export const ETH_CONTRACT_ADDRESS = '0x1111111111111111111111111111111111111111';
 
 export const COSMOS_ENDPOINT = 'http://LIGHT_CLIENT_ENDPOINT/';
+export const COSMOS_RPC_ENDPOINT = 'http://LIGHT_CLIENT_ENDPOINT/rpc/';
```
Comment (williamchong) [inline server/config/config.js:7]:
```suggestion
export const COSMOS_RPC_ENDPOINT = 'http://RPC_ENDPOINT/';
```

**Review by williamchong — CHANGES_REQUESTED** [review]

**Comment by williamchong** [comment]
please merge or rebase master thanks @shellyYG

File: constant/index.js [hunk 4]
```diff
@@ -6,4 +6,6 @@ export const BIGDIPPER_HOST = IS_TESTNET ? 'https://taipei.likecoin.bigdipper.li
 
 export const LIKECOIN_API_HOST = IS_TESTNET ? 'https://api.taipei.like.co' : 'https://api.like.co';
 
+export const CHAIN_ID = IS_TESTNET ? 'iscn-dev-chain-2' : 'likecoin-chain-sheungwan'; // TODO: change to mainnet ID
```
Comment (williamchong) [inline constant/index.js:9]: mainnet is `likecoin-mainnet-2`

File: assets/images/keplr.svg [hunk 5]
```diff
@@ -0,0 +1,159 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (williamchong) [inline assets/images/keplr.svg:1]: Can this file be minified via svgo?

**Review by williamchong — CHANGES_REQUESTED** [review]

File: assets/images/keplr.svg (hunk 5 above)
Comment (shellyYG) [reply assets/images/keplr.svg]: yes, looks still very clear.

**Review by williamchong — APPROVED** [review]
